modules/navier_stokes/src/physics/WCNSLinearFVFluidHeatTransferPhysics.C

```diff
@@ -277,11 +337,36 @@ WCNSLinearFVFluidHeatTransferPhysics::addEnergyOutletBC()
   for (const auto & outlet_bdy : outlet_boundaries)
   {
     const std::string bc_type = "LinearFVAdvectionDiffusionOutflowBC";
+    const auto variable_name = _solve_for_enthalpy ? _fluid_enthalpy_name : _fluid_temperature_name;
     InputParameters params = getFactory().getValidParams(bc_type);
     params.set<std::vector<BoundaryName>>("boundary") = {outlet_bdy};
     params.set<bool>("use_two_term_expansion") = getParam<bool>("energy_two_term_bc_expansion");
 
-    params.set<LinearVariableName>("variable") = _fluid_temperature_name;
-    getProblem().addLinearFVBC(bc_type, _fluid_temperature_name + "_" + outlet_bdy, params);
+    params.set<LinearVariableName>("variable") = variable_name;
+    getProblem().addLinearFVBC(bc_type, variable_name + "_" + outlet_bdy, params);
+  }
+}
+
+void
+WCNSLinearFVFluidHeatTransferPhysics::addMaterials()
+{
+  WCNSFVFluidHeatTransferPhysicsBase::addMaterials();
+
+  if (_solve_for_enthalpy)
+  {
+    // Define alpha, the diffusion coefficient when solving for enthalpy, on each block
+    for (unsigned int i = 0; i < _thermal_conductivity_name.size(); ++i)
+    {
+      const auto object_type = "ADParsedFunctorMaterial";
+      InputParameters params = getFactory().getValidParams(object_type);
+      assignBlocks(params, _blocks);
+      params.set<std::vector<std::string>>("functor_names") = {_thermal_conductivity_name[i],
+                                                               getSpecificHeatName()};
+      params.set<std::string>("expression") =
+          _thermal_conductivity_name[i] + "/" + getSpecificHeatName();
+      params.set<std::string>("property_name") = _thermal_conductivity_name[i] + "_by_cp";
+      getProblem().addMaterial(
+          object_type, prefix() + "alpha_from_" + _thermal_conductivity_name[i], params);
```

Comment:
alpha is k/cp/rho, not sure if there is a name for k/cp

Reply:
prefer to divide by rho then multiply by rho in the kernel?
or prefer to rename this to "alpha_times_rho" ?

Reply:
No, this is okay. I don't think there is a name for this.